include/hipSYCL/compiler/Frontend.hpp

```diff
@@ -171,7 +171,7 @@ inline std::string buildKernelName(clang::RecordDecl* D, clang::MangleContext *M
 }
 
 // Partially taken from CGCUDANV.cpp
-#if LLVM_VERSION_MAJOR >= 13
+#if LLVM_VERSION_MAJOR >= 13 || defined(HIPSYCL_NO_DEVICE_MANGLER)
```

Comment:
Shouldn't this be `&& !defined(HIPSYCL_NO_DEVICE_MANGLER)`?

Reply:
uhh... it should. yes. sorry